_posts/chall12/2017-05-22-caroline-turtle-hack.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+author: chall12
+title: "Caroline's Turtle Exercise"
+---
+
+
+Here's the program I'm embedding:
+
+
+<iframe src="https://trinket.io/embed/python/6c004e7fe9" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+
+Reflection:
+
+My idea for making a flower began on the first day of class when we learned to make circles. I first changed the background 
+color to light blue and created a rectangle along the base of the screen filled green to make grass. To begin the flower, I clumped two 
+circles together for each petal (it took me a while to get the size of the circle and their distances to work so that it didn't look 
+like a mass of yellow). Then I had to find the center of the flower to make the pollen cluster. Afterwords, I calculated the distance
+between the grass and the base of the flower to create the stem. It would have been smarter to create the stem first and draw the flower 
+over it, but I didn't think of that at first. Inserting the stem into the beginning of the code messed up the petal position, so 
+calculating its placement seemed like the easier alternative. Afterwords, I wanted to play with text some, so I added a few phrases 
+scattered around the flower in different colors.
+
+
+While this flower looks extremely simple, it actually took me over 2.5 hours to code. I had a really hard time with the petal placement, 
+and initially, I was going to have triangles attached at the base of two circles (similar to a heart) to make up each of the individual 
+petals. This turned out to be what took me a majority of an hour and I couldn't figure this out - could this be something we go over 
+in class? I get lost with whether to use right or left and which way the angle turns from where the turtle is.
```

Comment:
This would be a great thing to revisit once we have loops and functions.  If you could input a starting angle and have a function that makes a heart-shaped petal starting from that angle, you could call it in a loop to make a nice flower.  Keep this in the back of your mind.  You'll have the tools soon!